Makefile

```diff
@@ -1,6 +1,9 @@
 .PHONY: all proto
 
-all: proto
+all: proto go
```

Comment:
by convention, `all` usually only compiles things and doesn't create binaries, see:
https://www.gnu.org/prep/standards/html_node/Standard-Targets.html#Standard-Targets

Reply:
(tho i'm not sure i care that much)